src/typescript/frontend/src/lib/store/event-utils.ts

```diff
@@ -189,11 +189,12 @@ export const getLatestBars = (market: MarketStateValueType) => {
   return RESOLUTIONS_ARRAY.map((res) => ({ ...market[res]!.latestBar, period: res }));
 };
 
+const shouldKeep = (event: AnyEmojicoinEvent) => {
+  const time = getEmojicoinEventTime(event);
+  return time / 1000n > BigInt(new Date().getTime() - 60 * 1_000);
+};
```

Comment:
```suggestion
// Perhaps put in a `const.ts` file somewhere more appropriate
export const LOCAL_STORAGE_EXPIRY_TIME_MS = 60 * 1000;

const shouldKeep = (event: AnyEmojicoinEvent) => {
  const eventTimeMs = Number(getEmojicoinEventTime(event) / 1000n);
  const now = new Date().getTime();
  return eventTimeMs > now - LOCAL_STORAGE_EXPIRY_TIME_MS;
};
```

I know I'm being a little neurotic about these things so bare with me, but for the sake of readability I think we can split the logic up here